docs/ops/upgrading.md

```diff
@@ -183,15 +183,15 @@ In this step, we update the framework version of the cluster. What this basicall
 the Flink installation with the new version. This step can depend on how you are running Flink in your cluster (e.g. 
 standalone, on Mesos, ...).
 
-If you are unfamiliar with installing Flink in your cluster, please read the [deployment and cluster setup documentation]({{ site.baseurl }}/ops/deployment/cluster_setup.html).
+If you are unfamiliar with installing Flink in your cluster, please read the [clusters and deployment setup documentation]({{ site.baseurl }}/ops/deployment/cluster_setup.html).
```

Comment:
this is an unnecessary change.

Reply:
will revert back! I waned to make it similar to the `Clusters and Deployment` page title